src/app/covid-base/components/countries-table/sort.pipe.ts

```diff
@@ -2,20 +2,19 @@ import { Pipe, PipeTransform } from '@angular/core';
 import { ICountrData } from '../../../core/models/covid-base.models';
 
 @Pipe({
-  name: 'sortPipe'
+  name: 'sortPipe',
 })
 export class SortPipe implements PipeTransform {
-
   transform(array: ICountrData[], args: string): ICountrData[] {
     if (array !== undefined) {
       array.sort((a: any, b: any) => {
-          if ( a[args] < b[args] ){
-              return 1;
-          } else if ( a[args] > b[args] ) {
-              return -1;
-          } else {
-              return 0;
-          }
+        if (a[args] < b[args]) {
```

Comment:
These were fixed by this [commit](https://github.com/Anntol/covid-dashboard/commit/e35f16daab32ca8b0371f2d825296fc0d92c7c17#diff-6dc50a38c061b6ec39ab33cecaaf7c75e3685f3453e1ac3dd076cb31db81f69e)